src/Devices/Serial_TC.h

```diff
@@ -0,0 +1,85 @@
+// Year-Month-Day Format for DateTime
```

Comment:
This comment was part of our earlier discussion but doesn't seem fully relevant now. Instead it would be nice to have a comment that explained the class, what it wraps, and how to use it.